tests/testfunc.cpp

```diff
@@ -0,0 +1,4 @@
+int f()
```

Comment:
I tried to use plain.cpp instead but it seemed to hang when attempting to compile. Also it seems at a minimum we have to add at-least testmainfunc.cpp in order to test linking functionality.